great_expectations/cli/suite.py

```diff
@@ -318,6 +325,44 @@ def suite_new(suite, directory, empty, jupyter, view, batch_kwargs):
         raise e
 
 
+@suite.command(name="delete")
+@click.option("--suite", "-es", default=None, help="Expectation suite name.")
+@click.option(
+    "--directory",
+    "-y",
+    default=None,
+    help="Delete expectation suite.",
+)
+def suite_delete(suite, directory):
+    """Delete specified expectation suite from data_context expectation store.
+
+    Args:
+        expectation_suite_name: The name of the expectation_suite to delete
+
+    Returns:
+        True for Success and False for Failure.
+    """
+    try:
+        context = DataContext(directory)
+    except ge_exceptions.ConfigNotFoundError as err:
+        cli_message("<red>{}</red>".format(err.message))
+        return
```

Comment:
```suggestion
context = load_data_context_with_error_handling(directory)
```
This should save you time and be more complete. I'm trying to consolidate handling errors.

Reply:
Yup, nice one @Aylr Updated.